src/catch2/catch_session.cpp

```diff
@@ -296,6 +299,17 @@ namespace Catch {
         }
 
         CATCH_TRY {
+        if (!m_configData.rngSeedSpecified) {
+    if (const char* v = std::getenv("TEST_RANDOM_SEED")) {
```

Comment:
`std::getenv` is not supported on all platforms. See how other Bazel config options do this.